docs/guide/starting.rst

```diff
@@ -1,53 +1,28 @@
-============
-Starting off
-============
+=======================================================================
+Starting off: documentation best practices for developers
+=======================================================================
 
-So, you want to write the docs for your project?
+Want to write the docs for your project?
+You've come to the write [sic] place.
```

Comment:
I think [sic] is mostly for text copied from elsewhere? Not 100% sure tho. 

```suggestion
You've come to the *write* place.
```

Reply:
Right, a kinda "we know this is wrong, but didn't fix it cos it's a direct quote.